examples/svm/plot_rbf_parameters.py

```diff
@@ -85,7 +85,7 @@
 from sklearn.model_selection import StratifiedShuffleSplit
```

Comment:
As in the previous exercise, move the imports closer to their first usage.

Reply:
> As in the previous exercise, move the imports closer to their first usage.

If I revert my commit for this comment, every test runs correctly, not sure what did wrong, but please advise if I still should implement this, and if so how?